amundsen_application/static/templates/fragments/icons-dev.html

```diff
@@ -0,0 +1,10 @@
+<link rel="apple-touch-icon" sizes="180x180" href="/static/images/favicons/dev/apple-touch-icon.png">
```

Comment:
What do these particular cases cover? I'm not clear on why we need to generate this many images and when each one would get used.

Also, similar comment about anything that defines a color. What's the effect and if anything shouldn't these be Lyft and/or Amundsen theme colors?

Reply:
Yeah, the world of favicons and app icons and all that is really confusing, that's why it is better to just use a generator to create these and forget about the rest.

This is the generator I usully use: https://realfavicongenerator.net/. We just need to give it one icon and it creates the rest for you keeping the current best practices. If you are curious about the use of each of this icons, you can run the generator and learn more about them.

Reply:
I see I didn't pay attention to the `<!-- Generated with https://realfavicongenerator.net/ -->` comment. So all these files came from the generator, and you just made updates to the template and added environment variable logic?

Reply:
Yeah, basically!

Oh, I just added the comment :)